user-guide/howto/files/pxe/netboot_install_script.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
+#
+# The logic is simple:
+#  1. Download and extract USB image into /tmp/usbkey
+#  2. Run prompt-config.sh, which will create /usbkey on the zones pool
+#  3. Copy contents of /tmp/usbkey into /usbkey
+#
+# There is one little problem:
+#   The original prompt-config.sh does a reboot before it finishes =>
+#   => we need to change that before running the script.
+# 
+
+# Adjust this: download URL of the compute node USB image
+USB_URL="http://10.10.0.33/esdc-ce-cn-2.6.0.img"
```

Comment:
As this script is run only on clean install, you always want to use the latest image version. I recommend change this to:
```USB_URL="http://10.10.0.33/esdc-ce-cn-latest.img"```
And later, in zone image, the IP address will be automatically replaced by zone IP address using the cloud-init script.